tests/PHPUnit/formatXML.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
Is that test splitter still in use? Can see it is used somewhere, but there is actually only one test suite. Guess for the beginning it fine to have similar test splitting as on travis. We can later always look into splitting them up some more...

Reply:
@sgiehl I guess we can remove this file. it was used in the test, but I realize the max concurrency GitHub action allow is 20. And the exceed part has to wait for the available slot, it actually makes the whole process slower.